app/src/main/res/layout-land/admin_settings_dialog.xml

```diff
@@ -18,6 +18,10 @@
       android:id="@+id/input_pin"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
+      android:layout_marginStart="24dp"
+      android:layout_marginTop="36dp"
+      android:layout_marginEnd="24dp"
+      android:layout_marginBottom="36dp"
```

Comment:
This should be `20dp` or something near to this based on `sp to dp` conversion. The height that you are considering that height is between the ProfileInputView and the text of `Cancel` button (48dp clickable area). Instead we want height between the button and the ProfileInputView. Check this image for reference:

<img width="357" alt="Screenshot 2020-05-08 at 2 37 34 AM" src="https://user-images.githubusercontent.com/9396084/81344984-5a8f5c80-90d5-11ea-95ae-c0cdff17aadd.png">

Reply:
Ditto change for `portrait` mode

Reply:
Was having a conversation regarding that with chantel and she is updating mock. It will be updated soon.

Reply:
> Ditto change for `portrait` mode

updated this to 24dp as per https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/9603eda8-8d20-4377-9458-d5deec58b4d9/PC-NP-Admin-Settings-Dialog-Empty-/